ElectronClient/gui/StatusScreen.jsx

```diff
@@ -114,7 +136,14 @@ class StatusScreenComponent extends React.Component {
 				sectionsHtml.push(renderSectionHtml(i, section));
 			}
 
-			return <div>{sectionsHtml}</div>;
+			let retryAllHtml = renderRetryAllHtml(); // must be called after renderSectionHtml
```

Comment:
Please refactor the code so that this hack is not needed.

Reply:
Sure. BTW is there any coding style regarding `forEach` and `for(i < array.length)`? I notice a mixture of two in the project.

Reply:
I just refactored my code using `forEach` then, guess it's not a big deal 😂